apps/api/src/auth/auth.service.ts

```diff
@@ -426,4 +428,25 @@ export class AuthService {
           )
     }
   }
+
+  async deleteUser(keycloakId: string) {
+    const user = await this.personService.findOneByKeycloakId(keycloakId)
+
+    //Check and throw if user is a beneficiary, organizer or corporate profile
+    if (user?.beneficiaries?.length || user?.organizer || user?.companyId) {
```

Comment:
I would change this to:

   `if (user?.beneficiaries?.length > 0`

Seems more natural to read.

Reply:
I will adjust this later today.